default.nix

```
@@ -251,10 +251,10 @@ in emacs.overrideAttrs (esuper:
     cmd = ''
       wrapEmacs() {
           local -a wrapArgs=(
-              --set DOOMDIR ${doomDir}
+              --set-default DOOMDIR ${doomDir}
               --set NIX_DOOM_EMACS_BINARY $1
```

Might as well.. Sorting really irked me
```suggestion
              --set NIX_DOOM_EMACS_BINARY $1
              --set-default DOOMDIR ${doomDir}
```